app/src/lib/stores/app-store.ts

```diff
@@ -175,15 +175,15 @@ export class AppStore extends TypedBaseStore<IAppState> {
     string,
     RepositorySettingsStore
   >()
-  public readonly _gitHubUserStore: GitHubUserStore
-  private readonly _cloningRepositoriesStore: CloningRepositoriesStore
-  private readonly _emojiStore: EmojiStore
+  public readonly gitHubUserStore: GitHubUserStore
+  private readonly cloningRepositoriesStore: CloningRepositoriesStore
+  private readonly emojiStore: EmojiStore
   private readonly _issuesStore: IssuesStore
```

Comment:
`issuesStore`?

Reply:
Nope. it was `_issuesStore` before I started mucking around in this file. When I deleted the underscore,  TS complained about `issuesStore` already being defined. This is likely because the getter is the same name as the field (another reason why I prefer differentiating private members and public properties).